content/departments/product-engineering/engineering/enablement/dev-experience/index.md

```diff
@@ -52,6 +52,9 @@ To collaborate, we use the following:
 - Internal team channel in #dev-experience-internal
 - [GitHub planning board](https://github.com/orgs/sourcegraph/projects/212)
 - Daily updates via [Geekbot](https://app.geekbot.com/dashboard/standup/90468/view/insights) to #dev-experience-updates
+- Tools and languages update feed in #dev-experience-notification
+- GitHub issues and pull-requests feed in #dx-github-feed
```

Comment:
Actually are these for collaboration?

Reply:
This section lists what we use to _collaborate_ as a team, for things we own/manage/use to do our day-to-day-work maybe a separate resources section or an addition to [Responsibilities](https://handbook.sourcegraph.com/departments/product-engineering/engineering/enablement/dev-experience#responsibilities) is more apt

(sorry for the many comments!)

Reply:
Or maybe [Tech stack](https://handbook.sourcegraph.com/departments/product-engineering/engineering/enablement/dev-experience#tech-stack)?

Reply:
_Internal_ collaboration to be precise, but yes. I thought it'd be useful to document that. I just noticed for example that @taylorsperry wasn't on #dx-github-feed earlier today.

Reply:
I don't think this is for internal team collaboration, which I think are more communication tools and avenues we have to collaborate with each other, which include things like creating docs, issues, and messaging - this definitely falls under Tech stack and/or Responsibilities IMO